src/palace/manager/data_layer/circulation.py

```diff
@@ -319,12 +322,13 @@ def apply(
                     f"License pool status changing from {pool.status} to {self.status} for {pool.identifier!r}"
                 )
                 pool.status = self.status
+                changed_lp_status = True
 
-            if self.licenses is not None:
-                # If we have licenses set, use those to set our availability
+            if pool.type == LicensePoolType.AGGREGATED:
```

Comment:
Instead of using a proxy to choose how to do these updated (like checking if we have `self.licenses`) the apply behavior is now driven directly by the `pool.type`.